pkg/controller/disruption/disruption.go

```diff
@@ -706,8 +705,10 @@ func (dc *DisruptionController) trySync(ctx context.Context, pdb *policy.PodDisr
 	}
 	// We have unmamanged pods, instead of erroring and hotlooping in disruption controller, log and continue.
 	if len(unmanagedPods) > 0 {
-		klog.V(4).Infof("found unmanaged pods associated with this PDB: %v",
```

Comment:
Nit: I'd leave the log as was before, in case problems with connectivity with apiserver, at least we'll always have the log.

Reply:
I wonder if we shouldn't bump the verbosity to 2 or to 0? Also warnings are not supported anymore by go logr, and error may be too strong...

Reply:
We could, but that can be done in followup PR.

Reply:
found couple of more things, fixing in a followup https://github.com/kubernetes/kubernetes/pull/116256